test/features/discord.feature

```diff
@@ -1,101 +1,101 @@
-Feature: discord module
+# Feature: discord module
```

Comment:
As discussed I commented out the other tests, cause they would fail

Reply:
Sure 👍